models/attribute-definition/src/attribute-definition-draft/transformers.ts

```diff
@@ -0,0 +1,36 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type {
+  TAttributeDefinitionDraft,
+  TAttributeDefinitionDraftGraphql,
+} from './types';
+
+const transformers = {
+  default: Transformer<TAttributeDefinitionDraft, TAttributeDefinitionDraft>(
+    'default',
+    { buildFields: ['label', 'inputTip', 'type'] }
+  ),
+  rest: Transformer<TAttributeDefinitionDraft, TAttributeDefinitionDraft>(
+    'rest',
+    { buildFields: ['label', 'inputTip', 'type'] }
+  ),
+  graphql: Transformer<
+    TAttributeDefinitionDraft,
+    TAttributeDefinitionDraftGraphql
+  >('graphql', {
+    buildFields: ['type'],
```

Comment:
I feel like `label` and `inputTip` should be included here as well.

Reply:
🎸 fixed with (4df0ac4)[https://github.com/commercetools/test-data/pull/192/commits/4df0ac4aa3a69ee761f16e29c58c45606ef214c9]